<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimingWidget</class>
 <widget class="QWidget" name="TimingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>126</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_x">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_step1">
       <property name="text">
        <string>Time Index:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_t">
       <property name="maximum">
        <number>19</number>
       </property>
       <property name="sliderPosition">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_t"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_gradient">
       <property name="text">
        <string>Spacing (X):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_y">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_z">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Spacing (Y):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Spacing (Z):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_z">
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_t</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_t</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_t</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_t</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
